Makefile

```diff
@@ -35,6 +35,14 @@ endif
 wal-check: temp-install
 	$(prove_check)
 
+check: isolationcheck
```

Comment:
Can you remove `check` target? This target makes it impossible to run `check` with `USE_PGXS` variable. With this target you can run `check` only under the `contrib` directory.